idd/Energy+.idd.in

```diff
@@ -108793,10 +108793,21 @@ Output:JSON,
 Output:SQLite,
        \memo Output from EnergyPlus can be written to an SQLite format file.
        \unique-object
-  A1 ; \field Option Type
+  A1 , \field Option Type
        \type choice
        \key Simple
        \key SimpleAndTabular
+  A2 ; \field Unit Conversion for Tabular Data
+       \type choice
+	   \note Unit conversion option used when writing SQLite Tabular Data
+	   \note This option applies to TabularData and TabularDatawithString in the SQLite file
```

Comment:
Tab characters added in IDD here, need to be removed.

Reply:
Thanks!